doc/man1/openssl-dsaparam.pod.in

```diff
@@ -92,8 +93,14 @@ be handy during batch scripts and pipelines.
 =item I<numbits>
 
 This option specifies that a parameter set should be generated of size
-I<numbits>. It must be the last option. If this option is included then
-the input file (if any) is ignored.
+I<numbits>. If this option is included then the input file (if any) is
+ignored.
+
+=item I<numqbits>
+
+This option specifies that a parameter set should be generated with a subprime
```

Comment:
It is actually optional argument rather than an option. (Same for numbits.)

Reply:
"=item I<numbits>
This option specifies that a parameter ..."

Are you suggesting I should change both?

Reply:
Yep

Reply:
Done